src/App.js

```diff
@@ -1,14 +1,101 @@
 import React from 'react';
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm';
+
+
+const todos = [
+  {
+    task: 'Organize Garage',
+    id: 1528817077286,
+    completed: false
+  },
+
+  {
+    task: 'Go Grocery Shopping',
+    id: 1528817084358,
+    completed: false
+  },
+
+  {
+    task: 'Bake Cookies',
+    id: 1528817084359,
+    completed: false
+  }
+
+];
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+
+  constructor() {
+    super();
+    this.state = {
+      todoList: todos,
+    };
+  }
+
+  toggleItem = id => {
+    this.setState({
```

Comment:
I'd separate out the logic here to make it easier to read, but I think this probably runs faster